### DISCUSSION TITLE
You can really do anything you want.. with transferable skills!

### DISCUSSION THREAD

**Discussion by hijennyngo** [community/community#109396] [Programming Help]
The easiest way to summarize my professional journey would be to say that I successfully transitioned from a non-technical background to a technical career, but that’s not really fair to myself or others.

While growing up, I had plenty of exposure to computers 🖥️ and technology. I remember being 10 years old, and my dad bought me the parts for a new computer and refused to help me put it together. I whined a lot about needing help, and he wasn’t budging. So after a few hours and cutting up my hand from poorly removing a metal panel from the chassis, I finished putting it together myself. I also had to format the hard drive, install the operating system, and install the applications I needed.

Throughout my childhood, my brother and my dad were always the technical people in my family, where they loved customizing their computers. I didn’t really understand their obsession; I felt that a computer was just a tool to browse the Internet and chat with friends 💬.

When I got to college, my dad really wanted me to pursue a technical degree, but I looked at the requirements and thought: “No, thanks! This is way too many science 🧬 and math classes!” So I followed my own interest to pursue the creative track of an Advertising degree.

After graduating college, I worked at a newspaper 🗞️ supporting print advertising sales. I felt that my career growth was limited there as it was a declining industry, so I started looking into 🔍 transferable skills and what other roles I could do with those skills. I landed on an Executive Assistant position based on having these transferable skills: calendar management 📆, data entry, event planning, and travel booking ✈️. Most of these skills were acquired through my personal experience rather than work experience, but those advertising skills sure helped me sell these skills! 😂

While I was an Executive Assistant, I took opportunities to work on exploring transferable skills and exploring what roles I may want to pursue in the future. As much as I loved the Executive Assistant role, I felt that it wasn’t challenging enough for me.

I completed a Project Management certificate since a lot of my skills seemed to align with that role. The coursework resonated with me, so I wanted to witness some real experience with the role. I then shadowed a project manager within the company, ran a project with his supervision, and ended up deciding that it wasn’t a fit for me long-term either.

At the time, learning Python 🐍 was really popular, and one of the directors I supported challenged me to complete an Introduction to Python course on Codecademy. I was able to complete it and write some pretty inefficient code, but it was done! My “success story” was used as a motivator to get other technical folks within the organization to take the Python course as well.

I will say that social networking also helped me for this next transition. That same director was impressed that I was able to complete the Python course, write some Python code for him, and write an Excel script to clean up raw data to be visualized into a 3D graph 📈, so he offered me a role as a Support Engineer 👩🏻‍💻. However, I still believe that these transferable skills were also why he offered me this role: customer service, troubleshooting, problem solving, and helping and training others.

After this first switch into a technical role, I pursued a Sales Engineer position by leveraging a combination of skills from my previous experience: advertising sales, pitching advertising campaigns, copywriting, problem solving 💡, customer service, technical acumen, requirements gathering, etc. I also had to put together some mini PCs and help manage our physical lab infrastructure as part of the role, who knew my computer building 👩🏻‍🔧 skills could be put to use in this role! This role later evolved into a Customer Success Engineer role, where I focused on retaining customers prior to renewal by ensuring they adopted the purchased solution.

With my current role, the advantage of transferable skills came full circle, where I was able to leverage all the previous experience I had to become a Customer Success Manager . On a day-to-day basis, I leverage a combination of the skills I acquired from studying Advertising and Project Management, and my professional experience as Advertising Sales, Executive Assistant, Support Engineer, Sales Engineer, and Customer Success Engineer. Since I was also interested in improving customer telemetry data, I also completed a certificate in Data Science to understand conceptually how data scientists typically manipulate raw data to display meaningful insights 📊 about our customers.

In the end, I encourage you to look throughout your personal and professional experiences to find these transferable skills you may have overlooked and consider how they could be applicable to the next role you are looking to pursue. I also encourage you to be curious 🤔💭 and take courses that interest you, even if it’s not immediately relevant to your current aspirations as they may become useful in the future. Also, if you are unsure about your next move, I encourage you to shadow and/or find a stretch assignment to help confirm if you would want that role. You can really do anything you want.. with transferable skills!

**Comment by RAHULKRISHNAKR** [comment]
> The easiest way to summarize my professional journey would be to say that I successfully transitioned from a non-technical background to a technical career, but that’s not really fair to myself or others.
> 
> While growing up, I had plenty of exposure to computers 🖥️ and technology. I remember being 10 years old, and my dad bought me the parts for a new computer and refused to help me put it together. I whined a lot about needing help, and he wasn’t budging. So after a few hours and cutting up my hand from poorly removing a metal panel from the chassis, I finished putting it together myself. I also had to format the hard drive, install the operating system, and install the applications I needed.
> 
> Throughout my childhood, my brother and my dad were always the technical people in my family, where they loved customizing their computers. I didn’t really understand their obsession; I felt that a computer was just a tool to browse the Internet and chat with friends 💬.
> 
> When I got to college, my dad really wanted me to pursue a technical degree, but I looked at the requirements and thought: “No, thanks! This is way too many science 🧬 and math classes!” So I followed my own interest to pursue the creative track of an Advertising degree.
> 
> After graduating college, I worked at a newspaper 🗞️ supporting print advertising sales. I felt that my career growth was limited there as it was a declining industry, so I started looking into 🔍 transferable skills and what other roles I could do with those skills. I landed on an Executive Assistant position based on having these transferable skills: calendar management 📆, data entry, event planning, and travel booking ✈️. Most of these skills were acquired through my personal experience rather than work experience, but those advertising skills sure helped me sell these skills! 😂
> 
> While I was an Executive Assistant, I took opportunities to work on exploring transferable skills and exploring what roles I may want to pursue in the future. As much as I loved the Executive Assistant role, I felt that it wasn’t challenging enough for me.
> 
> I completed a Project Management certificate since a lot of my skills seemed to align with that role. The coursework resonated with me, so I wanted to witness some real experience with the role. I then shadowed a project manager within the company, ran a project with his supervision, and ended up deciding that it wasn’t a fit for me long-term either.
> 
> At the time, learning Python 🐍 was really popular, and one of the directors I supported challenged me to complete an Introduction to Python course on Codecademy. I was able to complete it and write some pretty inefficient code, but it was done! My “success story” was used as a motivator to get other technical folks within the organization to take the Python course as well.
> 
> I will say that social networking also helped me for this next transition. That same director was impressed that I was able to complete the Python course, write some Python code for him, and write an Excel script to clean up raw data to be visualized into a 3D graph 📈, so he offered me a role as a Support Engineer 👩🏻‍💻. However, I still believe that these transferable skills were also why he offered me this role: customer service, troubleshooting, problem solving, and helping and training others.
> 
> After this first switch into a technical role, I pursued a Sales Engineer position by leveraging a combination of skills from my previous experience: advertising sales, pitching advertising campaigns, copywriting, problem solving 💡, customer service, technical acumen, requirements gathering, etc. I also had to put together some mini PCs and help manage our physical lab infrastructure as part of the role, who knew my computer building 👩🏻‍🔧 skills could be put to use in this role! This role later evolved into a Customer Success Engineer role, where I focused on retaining customers prior to renewal by ensuring they adopted the purchased solution.
> 
> With my current role, the advantage of transferable skills came full circle, where I was able to leverage all the previous experience I had to become a Customer Success Manager . On a day-to-day basis, I leverage a combination of the skills I acquired from studying Advertising and Project Management, and my professional experience as Advertising Sales, Executive Assistant, Support Engineer, Sales Engineer, and Customer Success Engineer. Since I was also interested in improving customer telemetry data, I also completed a certificate in Data Science to understand conceptually how data scientists typically manipulate raw data to display meaningful insights 📊 about our customers.
> 
> In the end, I encourage you to look throughout your personal and professional experiences to find these transferable skills you may have overlooked and consider how they could be applicable to the next role you are looking to pursue. I also encourage you to be curious 🤔💭 and take courses that interest you, even if it’s not immediately relevant to your current aspirations as they may become useful in the future. Also, if you are unsure about your next move, I encourage you to shadow and/or find a stretch assignment to help confirm if you would want that role. You can really do anything you want.. with transferable skills!

👍👍

**Comment by Alrayan2025** [comment]
![img00](https://github.com/community/community/assets/160448672/88c0d012-c5c7-49e7-b041-f2d6d27e63ce)

**Reply by brotherbillo11** [reply]
AMERICA has TORTURED me and violated my Civil Rights my name is Brice K Reeves my address is 422  Natchez Trace Road Mathiston Mississippi 39752

**Comment by arabduur** [comment]
Thanks sis ! 
Makes me so motivated.

**Comment by Samge0** [comment]
great

**Comment by uditmahato** [comment]
I agree

**Comment by skliou3784** [comment]
**每一尔人都應平等對待**

**Comment by Mehran-Jalali** [comment]
Great job! Your efforts really bring motivation.☺

**Comment by ANIRUDHSINH01** [comment]
Hm

**Comment by mandarin37** [comment]
hhh

**Comment by arabduur** [comment]
Do u make react projects as well

**Reply by Dumickey** [reply]
Yes
